### PR TITLE
Improved three modules in showing more compacted info with new parameters

### PR DESCRIPTION
**Pull request by martindoublem**

### CODE REVIEW

File: modules/contrib/smartstatus.py [hunk 1]
```diff
@@ -29,6 +30,7 @@ def __init__(self, config, theme):
         self.devices = self.list_devices()
         self.display = self.parameter("display", "combined")
         self.drives = self.parameter("drives", "sda")
+        self.show_names = self.parameter("show_names", "True")
```
Comment (tobi-wan-kenobi) [inline modules/contrib/smartstatus.py:33]: For this, you can use `util.format.asbool()` to unify boolean handling - that automatically allows a user to use yes/no, true/false, 0/1 for the parameter.

File: modules/core/redshift.py [hunk 2]
```diff
@@ -74,6 +75,7 @@ def __init__(self, config, theme):
         super().__init__(config, theme, core.widget.Widget(self.text))
 
         self.__thread = None
+        self.transition_info = self.parameter("transition_info", "True")
```
Comment (tobi-wan-kenobi) [inline modules/core/redshift.py:78]: same here (`util.format.asbool()`)

File: modules/contrib/smartstatus.py (hunk 1 above)
Comment (martindoublem) [reply modules/contrib/smartstatus.py:33]: Man, I was wondering why the True False boolean was not working in your code!

**Comment by martindoublem** [comment]
Actually @tobi-wan-kenobi can you finish the last few changes, I am not sure how to use that util.format.asbool() in the conditional statements.

**Comment by tobi-wan-kenobi** [comment]
Changes pushed.

I also took the liberty of renaming "transition_info" in redshift to "show_transition", I think that captures the intent a bit clearer. I hope that is fine with you - if you have objects, please let me know, and I will gladly change the code again.

Thanks for the contribution!

**Comment by martindoublem** [comment]
You are always welcome, you know how much I like to use your status bar!

**Pull request merged**